### PR TITLE
[Hs-133] warning bottom sheet

### PR DESCRIPTION
**Pull request by dlvinh**

### CODE REVIEW

File: lib/data/data.dart [hunk 1]
```diff
@@ -12,6 +14,27 @@ enum Roles {
...
+      .firstWhere((element) => element.name == name, orElse: () => tenant);
+}
+
+enum PropertyTypes {
+  house,
+  apartment;
+
+  const PropertyTypes();
+
+  String getIconPath() {
```
Comment (hophongit98) [inline lib/data/data.dart:27]:
```suggestion
  String? getIconPath() {
```

File: lib/data/data.dart [hunk 2]
```diff
@@ -12,6 +14,27 @@ enum Roles {
...
+    switch (this) {
+      case house:
+        return Assets.images.house;
+      case apartment:
+        return Assets.images.apartment;
+      default:
+    }
+    return "No image path";
```
Comment (hophongit98) [inline lib/data/data.dart:35]:
```suggestion
    return null;
```

File: lib/features/add_property/view/add_property_view.dart [hunk 3]
```diff
@@ -1,14 +1,154 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
```
Comment (hophongit98) [inline lib/features/add_property/view/add_property_view.dart:2]: Oops. the scope of this HS-133 focus on warning_lost data. you don't have to add UI for add property view.

Comment (hophongit98) [reply lib/features/add_property/view/add_property_view.dart:2]: revert this class

**Review by hellomentorplus — APPROVED** [review]

**Review by suesitran — APPROVED** [review]

**Pull request merged**